src/components/templates/BottomSheetNotificationSettings/index.tsx

```diff
@@ -158,22 +158,22 @@ export const BottomSheetNotificationSettings = forwardRef<
           />
           <Spacer value={verticalScale(24)} />
           {/* Transaction Alerts */}
-          <Button
-            onPress={() =>
-              onSettingsValueChange(
-                'transactionAlerts',
-                !localNotificationSettings.transactionAlerts
-              )
-            }
-          >
-            <Row alignItems="center" justifyContent="space-between">
-              <Title>Transaction alert</Title>
+          <Row alignItems="center" justifyContent="space-between">
+            <Title>Transaction alert</Title>
+            <Pressable
```

Comment:
onValueChange prop of Switch can be used instead of Pressable

Reply:
also can we remove **disabled** prop of Switch than?

Reply:
Yes